docs/appendices/release-notes/unreleased.rst

```diff
@@ -117,6 +117,12 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
+- Fixed an issue that caused ``DROP TABLE IF EXISTS`` to wrongly return ``1``
```

Comment:
minor: we could mention `1 row affected or SQLParseExceptions depending on user privileges`

Reply:
thx! added.